portal-web/test/functional/com/liferay/portalweb/paths/portlet/forms/FormFields.path

```diff
@@ -188,6 +198,11 @@
 	<td>//div[contains(@class,'form-group')]//span[contains(@class,'form-text') and contains(text(),'${value1}')] | //div[contains(@data-field-name,'${key_fieldName}')]//p[@class='liferay-ddm-form-field-tip'] | //div[contains(@data-field-name,'${key_fieldName}')]//span[contains(@class, 'form-text')]</td>
 	<td></td>
 </tr>
+<tr>
+    <td>HELP_TEXT_AT_APP_DEPLOYED</td>
```

Comment:
@leilanyaragao, please rename to `HELP_TEXT` or `HELP_TEXT_VALUE` only. `APP_DEPLOYED` isn't a term from Forms. :wink: 

Thanks.